### PR TITLE
Ensure only valid KE phone numbers are sent to paygops - SER-2149

### PR DESCRIPTION
**Pull request by victorjbassey**

### CODE REVIEW

File: src/main/java/org/mifos/connector/ams/paygops/paygopsDTO/PaygopsRequestDTO.java [hunk 1]
```diff
@@ -63,4 +68,29 @@ public String toString() {
...
+     * @return true if the phone number is valid
+     */
+    private boolean isValidPhoneNumber(String phoneNumber) {
+        if (phoneNumber == null) return false;
+        String number = phoneNumber.trim();
+        if (number.length() == 0) return false;
+        try {
+            String fullPhoneNumber = number.startsWith("+") ? number : "+" + number;
+            PhoneNumberUtil phoneUtil = PhoneNumberUtil.getInstance();
+            Phonenumber.PhoneNumber phone = phoneUtil.parse(fullPhoneNumber, Phonenumber.PhoneNumber.CountryCodeSource.UNSPECIFIED.name());
```
Comment (gilclefes) [inline src/main/java/org/mifos/connector/ams/paygops/paygopsDTO/PaygopsRequestDTO.java]: @victorjbassey can we validate that this is a Kenyan number or number, I think even if it is a valid phone number but not Kenya, we should not send it to paygops.

**Review by shejasabin — DISMISSED** [review]

**Review by gilclefes — COMMENTED** [review]
@victorjbassey can we validate that this is a Kenyan number or number, I think even if it is a valid phone number but not Kenya, we should not send it to paygops.

File: src/main/java/org/mifos/connector/ams/paygops/paygopsDTO/PaygopsRequestDTO.java (hunk 1 above)
Comment (victorjbassey) [reply src/main/java/org/mifos/connector/ams/paygops/paygopsDTO/PaygopsRequestDTO.java]: Yeah, we can do that. 
That means we don't expect paygops to be used for other countries in future, right?

Comment (gilclefes) [reply src/main/java/org/mifos/connector/ams/paygops/paygopsDTO/PaygopsRequestDTO.java]: Yes, at least for now.

**Review by shejasabin — APPROVED** [review]

**Review by gilclefes — APPROVED** [review]
looks good to me

**Pull request merged**